python/ray/node.py

```diff
@@ -26,7 +26,23 @@
 logger = logging.getLogger(__name__)
 
 SESSION_LATEST = "session_latest"
-NUMBER_OF_PORT_RETRIES = 40
+NUM_PORT_RETRIES = 40
+NUM_REDIS_GET_RETRIES = 20
```

Comment:
40s retry seems very long considering that the key should get added soon after startup, but this shouldn't happen under normal operation so I'm not too concerned.